### PR TITLE
GRAU-249

### PR DESCRIPTION
**Pull request by DanMaguireData**

### CODE REVIEW

File: .storybook/storybook.requires.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {start} from '@storybook/react-native';
 const normalizedStories = [
   {
     titlePrefix: '',
-    directory: './.storybook/stories',
+    directory: '../.storybook/stories',
```
Comment (DANLENEHAN) [inline .storybook/storybook.requires.ts:8]: These are in the same dir aren't they?

Comment (DanMaguireData) [reply .storybook/storybook.requires.ts]: Yeah they are, no clue how that happened, reverted now

File: src/screens/track/mood/MoodTag.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ import ButtonComponent from '@components/buttons/ButtonComponent';
 // Types
 import {ScreenProps} from '@screens/Types';
 import {Mood} from '@screens/track/mood/Types';
+import {MoodTagGroups} from './Types';
```
Comment (DANLENEHAN) [inline src/screens/track/mood/MoodTag.tsx:20]: Relative Import 👮‍♂️

File: src/screens/track/mood/MoodTag.tsx [hunk 3]
```diff
@@ -56,17 +57,36 @@ const MoodTagScreen: React.FC<any> = ({
...
   const [note, setNote] = useState('');
   const [isCollapsed, setIsCollapsed] = useState(false);
 
+  // State for the mood tags
+  const [selectedMoodTags, setMoodTags] = useState<{[key: string]: string[]}>(
+    {},
+  );
+
+  const selectTag = (category: string, tag_id: string) => {
+    console.log('category', category, tag_id);
```
Comment (DANLENEHAN) [inline src/screens/track/mood/MoodTag.tsx:79]: Console log 🚓

**Review by DANLENEHAN — APPROVED** [review]

**Pull request merged**